testing/metaopt/test_automl.py

```diff
@@ -0,0 +1,163 @@
+import numpy as np
+from sklearn import linear_model
+from sklearn.metrics import mean_squared_error
```

Comment:
Remove sklearn imports from this file. Sklearn imports should be minimal in case we may want to remove the dependency later on. 

Let's try to import scikit-learn only in a few examples or tests, and in the package `neuraxle.steps.sklearn` for instance.

Reply:
In the worst case: document that a possible function to use as a metric in the class AutoML is the `from sklearn.metrics import mean_squared_error` function.